### PR TITLE
FEATURE: Fr and En Localization (closes #2)

### LINKED ISSUE #2: French and english localization
- On the header and footer:
   - change "Concordancier" (fr) and "Concorder" (en) into "Concordance" in both languages. We speak about the object used by people rather than about the computer program.
   - Change "Copyrights" (fr) into "Droits d'auteur".
- On the copyright page:
   - The french localization seems to be missing. Texts can be found on the original [mockup](https://github.com/benel/TraduXio/wiki/copyrights.png).
   - "Report an on-line abuse" (en) is not a correct translation of "Signaler une mise en ligne abusive" (fr). It should be instead something like "Report a copyright abuse".
  - In "Publication of protected work non-comparable to a short quotation" (en), "non-comparable" does not seem to be a correct translation for "non-assimilable" (fr) in a legal context. We should ask a native speaker.

### PR DESCRIPTION
**Pull request by adeprez**

### CODE REVIEW

File: couchdb/_attachments/traduxio.js [hunk 1]
```diff
@@ -2,8 +2,10 @@ $.fn.concordancify = function() {
   
   default_language=$("body").data("language") || currentLanguage;
   default_query=$("body").data("query") || "";
+  
+  var research = $("#nav").data("i_research");
 
-  this.append('<input id="query" type="search" results="5" name="query" placeholder="Rechercher" value="'
+  this.append('<input id="query" type="search" results="5" name="query" placeholder="' + research + '" value="'
```
Comment (benel) [inline couchdb/_attachments/traduxio.js]: In English, it would be `search` instead of `research`.

File: couchdb/_attachments/traduxio.js [hunk 2]
```diff
@@ -44,7 +46,8 @@ var currentLanguage='fr';
 
 function getLanguageName(id,target) {
   var result=id;
-  target=target || currentLanguage;
+  target=target || default_language;
+  console.log(target);
```
Comment (benel) [inline couchdb/_attachments/traduxio.js]: Debug messages shouldn't be published.

**Comment by franck-eyraud** [comment]
Je signale que ces modifications sont faites sur une très vieille version qui ne correspond pas à ce qui est en production...

File: couchdb/i18n.json [hunk 3]
```diff
@@ -0,0 +1,130 @@
+{
+  "en": {
+    "i_analyze_compare": "Analyze and compare",
+	"i_participative_translation": "Participative cultural text translation",
```
Comment (benel) [inline couchdb/i18n.json]: Indentation should be done with spaces rather than with tabs.

**Comment by benel** [comment]
@franck-eyraud The problem is that we don't have much choice: the `collaborative` branch has never been validated. This semester, a team of students will retro-design, validate and integrate it feature by feature.

File: couchdb/shows/work.js [hunk 4]
```diff
@@ -41,7 +42,8 @@ function(o, req) {
     headers: [],
     units: [],
     raw:[],
-    rows:[]
+    rows:[],
+	i18n: localized()
```
Comment (benel) [inline couchdb/shows/work.js]: Indentation should be done with spaces rather than with tabs.

**Comment by franck-eyraud** [comment]
Ok, sure, I get the point. Good work, and if you need help just ask!

File: couchdb/_attachments/languages.json [hunk 5]
```diff
@@ -35,7 +35,7 @@
 "dv":{"en":"Divehi","dv":"‫ދިވެހި","fr":"Dhivehi"},
 "dz":{"en":"Dzongkha","dz":"རྫོང་ཁ","fr":"Dzongkha"},
 "ee":{"en":"Ewe","ee":"Ɛʋɛgbɛ","fr":"Ewe"},
-"el":{"en":"Greek","el":"Ελληνικά","fr":"Grec"},
+"el":{"en":"Greek","el":"Ελληνικά","fr":"Grec moderne"},
```
Comment (benel) [inline couchdb/_attachments/languages.json]: En fait, c'est volontairement que l'on avait mis "Grec" et non "Grec moderne".
Cette norme, avec juste deux lettres, ne fait pas la différence.

**Comment by GAGNAIRE-Romain** [comment]
## Possible scenario

1. The user may choose his preferred languages.
2. The user can visit either the [traduxio website](traduxio.hypertopic.org) or the [traduxio testing website](traduxio.sandbox.hypertopic.org).
3. The interface will adapt to the user's preferred languages based on the ordering he defined on its browser. If there is no suitable localization available, the interface will use a fallback language (_most likely english_).

## Improvement ideas

- use some kind of button or menu to change the interface language instead of having to modify browser settings.

## Tutorial : How to setup preferred languages on Chrome

1. Open Chome's Settings menu, and click on `show advanced settings`
settings00

2. Click on `choose languages`
settings02

3. You can add as many languages as you want using the `add` button
settings03

4. Here is an example showing the addition of Korean
settings04

5. You can then sort languages as you want
settings05

File: couchdb/i18n.json [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    "i_delete_text_and_translations": "Supprimer le texte et toutes ses traductions",
+    "i_hide": "Cacher",
+    "i_edit": "Éditer",
+    "i_edit_license": "Éditer la licence",
+    "i_all_rights_reserved": "Tous droits réservés",
+    
+    "i_search": "Rechercher",
+    "i_confirm_delete": "La suppression est irréversible. Continuer ?",
+    "i_read": "Lire",
+    "i_year": "Année",
```
Comment (SAMIEMEHEZA) [inline couchdb/i18n.json]: Les élements "Date, année, ou siècle de la traduction" ne sont plus diponibles.
On ne retrouve que Année.

Comment (benel) [inline couchdb/i18n.json]: Autrement dit, il y a deux problèmes : 
- il faut indiquer que ce n'est pas forcément une "année" (l'année précise est inconnue pour les textes anciens),
- il faut distinguer explicitement la datation de la VO et celles de ses traductions (c'est le cas aussi pour d'autres champs).

**Pull request closed without merging**